packages/eslint-config-vtex/rules/style.js

```diff
@@ -3,7 +3,9 @@ module.exports = {
     // Allow brace-less single-line if, else if, else, for, while, or do, while still enforcing the use of curly braces for other instances.
     // ! Overrides definition on 'eslint-config-prettier'
```

Comment:
```suggestion
```

Reply:
Also, it only kind of "conflitcs" with prettier. If prettier decides to insert a line break before an `if() expression;` line is too long, I'd say that adding curlies are the correct thing to do

Reply:
```js
if (cart.items && cart.items[0] && cart.items[0].quantity === 0)
  updateCart(cart);
```

personally, should be

```js
if (cart.items && cart.items[0] && cart.items[0].quantity === 0) {
  updateCart(cart);
}
```

Reply:
I agree with you, maybe this is something eslint could improve to apply the multi-line rule only when the if + expression doesn't reach the max-len.